packages/concerto-cli/index.js

```diff
@@ -20,13 +20,18 @@ const Commands = require('./lib/commands');
 
 require('yargs')
     .scriptName('concerto')
-    .usage('$0 <cmd> [args]')
+    .usage('$0 <cmd> [manager] [args]')
     .command('validate', 'validate JSON against model files', (yargs) => {
         yargs.option('sample', {
             describe: 'sample JSON to validate',
             type: 'string',
             default: 'sample.json'
         });
+        yargs.option('ctoSystem', {
+            describe: 'system model to be used',
+            type: 'string'
+            default: 'org.accordproject.base.cto'
```

Comment:
I'm unclear how the default can be a namespace, I was expecting it to be consistent with the option which should pass a file (or maybe a URL or either?) in a way similar to how other CTO files are passed on the command line.

Without a default, the value will be null which might be sufficient to decide whether a system model has been passed or not.